src/testing/docker/travis_docker_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+docker build -t fossology/test . 
+container_id=`docker run -p 8081:80 -d fossology/test`
+ip=`docker inspect --format '{{ .NetworkSettings.IPAddress }}' $container_id | awk "NR==1{print;exit}"`
+docker ps | grep -q fossology/test
+curl -L -I http://$ip:8081/repo/ | grep -q "200 OK"
```

Comment:
You should also grep in the full response for some content expected on the fossology page. If the composer installation fails, apache still serves an empty page with response code 200